.github/workflows/pr-labeler.yml

```diff
@@ -0,0 +1,93 @@
+name: Pull Request Labeler
+# This workflows works in conjunction with the release.production.yml workflow to automate release notes based on labels.
+
+on:
+  pull_request:
```

Comment:
doesn't this require additional configuration like `on close`, `on synchronize` et cetera?

Reply:
> By default, a workflow only runs when a pull_request event's activity type is opened, synchronize, or reopened

They should be enough